bash_completion.sh

```diff
@@ -128,11 +128,32 @@ _buildtest ()
 {
   local cur="${COMP_WORDS[COMP_CWORD]}"
   local prev="${COMP_WORDS[COMP_CWORD-1]}"
+  declare -i offset="0"
 
   COMPREPLY=()   # Array variable storing the possible completions.
 
-  next=${COMP_WORDS[1]}
+  declare -a buildtest_opts=("--color" "--config" "--debug" "--editor" "--help" "--helpcolor" "--logpath" "--loglevel" "--print-log" "--no-color" "--report" "--version" "--view-log" "-c" "-d" "-h" "-l" "-p" "-r" "-V") 
+  
+  for command in "${COMP_WORDS[@]}"
+  do
+    for element in "${buildtest_opts[@]}"
+    do
+
+        if [[ "$command" == "$element" ]]; then
+          
+          if [[ "$command" == "--color" ]] || [[ "$command" == "--config" ]] || [[ "$command" == "-c" ]] || [[ "$command" == "--report" ]] || [[ "$command" == "-r" ]];
```

Comment:
i think i understand what this is trying to do its catching the edge case where we are offset by 2 due to input argument

```
buildtest --color <COLOR> <sub-command>
buildtest --report <report_file> sub-command>
```

There are few other edge-cases that haven't been accounted for, in this one we could have more offset

```
buildtest -d --loglevel <LOG_LEVEL> <sub-command>
buildtest --editor <editor> bc edit
```

Reply:
tab completion works for `--loglevel` 
![Screen Shot 2023-04-07 at 10 34 19 AM](https://user-images.githubusercontent.com/12942230/230626601-e8ae3692-5d49-4bc0-9bd1-3e9ba0508a4f.png)

But this doesn't work in conjuction with a sub-command
![Screen Shot 2023-04-07 at 10 34 45 AM](https://user-images.githubusercontent.com/12942230/230626648-f861b982-ddb3-46ff-8569-60fa484a5cc4.png)

Reply:
Thanks @shahzebsiddiqui for catching that. I have added the edge cases now.

<img width="1302" alt="Screen Shot 2023-04-07 at 9 38 00 AM" src="https://user-images.githubusercontent.com/35829130/230645543-0c02d3ed-a8da-4692-b4ff-f3d077f23e3e.png">